CHANGELOG.md

```diff
@@ -65,6 +65,26 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+#### Component JavaScript modules have a unique name when imported individually
```

Comment:
I _think_ we've sort of settled on using a description of 'what the user needs to do' for breaking changes (for example 'Make sure character counts still work as expected') – is there an equivalent we could use for this?

I also think we might want to list this under 'Fixes' as well, as it feels like a bug?

Reply:
@EoinShaughnessy I've had a go at the release notes for this (including Ollie's suggestion here), but I think they could do with some love! Would it be helpful if I put some time in to talk through it together?

Reply:
@vanitabarrett That'd be great, thanks!

Reply:
@EoinShaughnessy Let's have a chat about it at our catch-up tomorrow 👍